tests/e2e/configurer/chain/commands.go

```diff
@@ -85,8 +85,9 @@ func (n *NodeConfig) CreateConcentratedPool(from, denom1, denom2 string, tickSpa
 // token{0,1} - liquidity to create position with
 func (n *NodeConfig) CreateConcentratedPosition(from, lowerTick, upperTick string, token0, token1 string, token0MinAmt, token1MinAmt int64, poolId uint64) uint64 {
 	n.LogActionF("creating concentrated position")
-
-	cmd := []string{"osmosisd", "tx", "concentratedliquidity", "create-position", lowerTick, upperTick, token0, token1, fmt.Sprintf("%d", token0MinAmt), fmt.Sprintf("%d", token1MinAmt), fmt.Sprintf("--from=%s", from), fmt.Sprintf("--pool-id=%d", poolId), "-o json"}
+	// gas = 50,000 because e2e  default to 40,000, we hardcoded extra 10k gas to initialize tick
+	// fees = 1250 (because 50,000 * 0.0025 = 1250)
```

Comment:
Shouldn't we be initializing two ticks per position?

Reply:
i confirmed in `UpdatePosition` unit test to make sure we are using 20k gas while initializing test here's the output 
```
with consume gas 
=== RUN   TestKeeperTestSuite/TestUpdatePosition/update_existing_position_with_positive_amount
BEFORE  INITIALIZE TICK (Lower & Upper) :  345134
AFTER  INITIALIZE TICK (Lower & Upper) GAS:  396742

without consume gas 
=== RUN   TestKeeperTestSuite/TestUpdatePosition/update_existing_position_with_positive_amount
BEFORE  INITIALIZE TICK (Lower & Upper) GAS:  345134
AFTER INITIALIZE TICK (Lower & Upper) GAS:  376742
```

maybe in e2e the fees were too low to begin with?

Reply:
Discussed offline to not block on this but keep this open question in mind. Thanks for looking into it @stackman27